lib/index.js

```diff
@@ -288,9 +288,7 @@ var MultiMenuWrapper = exports.MultiMenuWrapper = function (_Component2) {
           className: (0, _classnames2.default)('multi-menu-wrapper', 'expand-' + expandDirection, 'expand-' + verticalExpand, className, { 'multi-menu-wrapper-attached': isStyledMenu }),
           style: position
         },
-        _react2.default.createElement(_reactDocumentEvents2.default, {
```

Comment:
It doesn't look like the gitignore is working on lib files like you expected it to.

Reply:
Yeah, not totally sure here. I tried to add the folder to .gitignore but wasn't successful. I don't think it was ever on there.

Reply:
Try this @scott-massey https://stackoverflow.com/a/30360954

Reply:
Done